ime/app/src/main/res/xml/popup_kaomoji.xml

```diff
@@ -1,49 +1,132 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <Keyboard xmlns:android="http://schemas.android.com/apk/res/android"
-          android:keyWidth="15%p"
-          android:keyHeight="@integer/key_normal_height">
+        android:keyWidth="15%p"
```

Comment:
missing namespace `ask`.
add `xmlns:ask="http://schemas.android.com/apk/res-auto"`

Reply:
Hopefully got it in there right.

Reply:
seems right